src/package.rs

```diff
@@ -66,17 +59,25 @@ impl Package {
         }
     }
 
+    /// License of the package
     pub fn license(&self) -> &str {
         &self.license
     }
 
+    /// Succinct description of the package
     pub fn summary(&self) -> &str {
         &self.summary
     }
 
+    /// Longer description of the package
     pub fn description(&self) -> &str {
         &self.description
     }
+
+    /// Buildtime of the package
+    pub fn buildtime(&self) -> time::SystemTime {
+        time::SystemTime::UNIX_EPOCH + time::Duration::new(self.buildtime as u64, 0)
```

Comment:
I'm guessing the `u64` conversion here probably won't be an issue in practice, but if you wanted to be a bit more on the safe side you could do:

```suggestion
        let buildtime = u64::try_from(self.buildtime).expect("negative build time");
        time::SystemTime::UNIX_EPOCH + time::Duration::new(buildtime, 0)
```

Reply:
Yes. Done.